app/controllers/static_page_controller.rb

```diff
@@ -0,0 +1,9 @@
+class StaticPageController < ApplicationController
+  def home
+  end
```

Comment:
```suggestion
  def home; end
```

sai coding convention em nhé